### PR TITLE
Dev-6379 Tabs Component

### PR DESCRIPTION
**Pull request by jonhill13**
### Author
- [x] Story Book has been rebuilt (new build artifacts in `/docs`)
- [x] Components have been rebuilt (new build artifacts in `/dist`)

👆These can be done by executing `npm run build` and committing the build artifacts.

- [x] All new components have been exported in `/index.js` `(if applicable)`
- [x] Design Review Complete
- [x] Relevant Draft release notes have been updated describing these changes ([use this template](https://github.com/fedspendingtransparency/data-transparency-ui/blob/master/release_notes_template.md) to document your changes) `(if applicable)`
- [x] Integration Status Updated on relevant documentation page `(if applicable)`
- [x] `index.d.ts` updated with new prop types defined for new components. `(if applicable)`

### Reviewer
- [x] Code review

### CODE REVIEW

File: .storybook/stories/tabs.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            label: 'Dates'
+          }
+        ]}
+        counts={{ name: 101, dates: 1002 }}
+        hideCounts={boolean('hideCounts', false)} />
+      </TabsWrapper>
+    </Story>
+</Preview>
+
+<Props of={Tabs} />
```
Comment (MaxwellKendall) [inline .storybook/stories/tabs.stories.mdx:76]: Lets start standardizing these pages. I like to have the props table at the top.

you can also throw in any kind of mark down you want, even inside the Preview tag, or just above it, whatever.

Comment (MaxwellKendall) [reply .storybook/stories/tabs.stories.mdx:76]: We should also have a checklist for where this stuff is used. 

- Advanced search
- Covid Profile
- Award Page
- Keyword search

Or maybe just say "this component is not currently used on the website." basically we just want a status report.

File: styles/components/_tabs.scss [hunk 2]
```diff
@@ -0,0 +1,105 @@
+@import "../global/index";
+
+.table-types {
+  border-bottom: 2px solid $color-gray-light;
+  padding-left: rem(15);
+  width: 100%;
+  // display: none;
```
Comment (MaxwellKendall) [inline styles/components/_tabs.scss:7]: should we delete this?

File: .storybook/stories/tabs.stories.mdx (hunk 1 above)
Comment (jonhill13) [reply .storybook/stories/tabs.stories.mdx:76]: Yeah, we should discuss how we can improve this library and the documentation.

**Review by MaxwellKendall — APPROVED** [review]

**Pull request merged**